Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -18,6 +18,43 @@ namespace Xamarin.Forms.Platform.Android
 	public static class ResourceManager
 	{
 		const string _drawableDefType = "drawable";
+		static ImageCache _lruCache = null;
+		static object _lruCacheHandle = new object();
+
+		static ImageCache GetCache()
+		{
+			if (_lruCache != null)
+				return _lruCache;
+
+			lock(_lruCacheHandle)
+			{
+				if (_lruCache == null)
+					_lruCache = new ImageCache();
+			}
+
+			return _lruCache;
+		}
+
+		class ImageCache : global::Android.Util.LruCache
+		{
+
+			static int getCacheSize()
+			{
+				// taken from lru cache docs
```

Comment:
Can we get a link to the docs where you found this? Easier for future maintainers to understand what's happening.